pyxlsb/cell.py

```diff
@@ -13,19 +13,43 @@ def c(self):
     def v(self):
         return self.value
 
+    @property
+    def value_conv(self):
+        dtype = self.dtype
+        if dtype == "datetime":
+            return self.row_cls.sheet.workbook.convert_date(self.value)
+        elif dtype == "string":
+            return str(self.value)
+        else:
+            return self.value
+
+    @property
+    def number_format(self):
+        return self.row_cls.sheet.workbook.styles.get_number_format(self.style_num)
+
+    @property
+    def dtype(self):
+        return self.row_cls.sheet.workbook.styles.get_dtype(self.style_num)
+
     @property
     def f(self):
         return self.formula
 
+    @property
+    def s(self):
+        return self.style
+
 class Cell(DeprecatedCellMixin):
-    __slots__ = ('row', 'col', 'value', 'formula')
+    __slots__ = ('row_cls', 'row', 'col', 'value', 'formula', 'style_num')
 
-    def __init__(self, row, col, value=None, formula=None):
-        self.row = row
+    def __init__(self, row_cls, col, value=None, formula=None, style_num=None):
+        self.row_cls = row_cls
```

Comment:
I think row/col should be coordinates of this cell. So if we need to store which row contain this cell, this should be variable with understandable name